src/modules/common/interfaces/filemeta.interface.ts

```diff
@@ -0,0 +1,10 @@
+export interface FileMeta extends Document {
+  readonly fieldname: string;
```

Comment:
What is that?

Reply:
Apparently that's the kind of file, for example `image`, `video`, etc... All these fields are provided by the node package that uploads the file